### ISSUE TITLE
New Multimesh dropdown not working in editor in MacOS in 3.5.stable version

### ISSUE THREAD

**Issue by vickylance** [godotengine/godot#67476]
### Godot version

3.5.stable

### System information

M1 MacBook Pro - 16 inch

### Issue description

When I add a new MultiMeshInstance to the scene tree and then click on it and create a new MultiMesh. And then click on it, I am getting an empty dropdown.
I have attached a sample video.

https://user-images.githubusercontent.com/5861833/196023195-f9d79ad5-eb9f-4635-a0f6-4ede318ffe9b.mov

### Steps to reproduce

current behaviour
1. Add a new multimeshinstance in godot 3.5 stable in macos
2. create a new multimesh
3. click on the multimesh
4. No dropdown will appear

expected behaviour
4. dropdown properties of new multimesh should appear

**Comment by Calinou** [comment]
I can't reproduce this on 3.6.beta 2b1544b4f:

![image](https://user-images.githubusercontent.com/180032/196051742-7ebfa5f6-f72d-4107-8b94-d65f0cbf2275.png)

**Comment by vickylance** [comment]
@Calinou  I re-downloaded 3.5.1 stable and tried again on an empty MultiMeshInstance2D its still not working.
Where can I download 3.6.beta? Also is it possible for anyone else to test on macos preferable M1 macos?

<img width="291" alt="image" src="https://user-images.githubusercontent.com/5861833/196165194-65aaa593-aafd-43d6-89c7-f5c154446545.png">

**Comment by vickylance** [comment]
Not working in Godot 4 beta 2 as well
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/5861833/196166076-afc23359-c6ce-4810-b9d5-9ff05352c370.png">

**Comment by kleonc** [comment]
I can reproduce it if I'd... remove `MultiMesh` from the editor features. :smile: (both 3.x/4.0)
![Gvn6JyzXdJ](https://user-images.githubusercontent.com/9283098/196184094-fc72ac5d-2350-4622-8b43-4eed52f9510a.png)
![EN18mEE52l](https://user-images.githubusercontent.com/9283098/196184126-d5d1b91d-5517-4760-b4c1-76ffe2555276.png)
![lHfJEI8THK](https://user-images.githubusercontent.com/9283098/196184288-23b718e7-6340-45b4-8179-aafcedaf71e2.png)


@vickylance Are you sure you haven't done that or something similar?

**Comment by vickylance** [comment]
@kleonc  Yes you are absolutely right. I had it disabled it thinking its a 3d only feature 😓 

<img width="1479" alt="image" src="https://user-images.githubusercontent.com/5861833/196185683-f397b80a-779f-4e80-965d-e9065c67dbc5.png">

Thanks you so much for finding the issue.

**Comment by kleonc** [comment]
Anyway, I think it's not solved properly. In case `MultiMesh` (or any resource) is disabled then it probably shouldn't show up as one of the options in this menu in the first place:
![9lsMDhSgOA](https://user-images.githubusercontent.com/9283098/196190209-dee996c1-2a8b-496e-b37f-ca97d485a0d3.png)
And when trying to inspect such disabled `MultiMesh` (which could have been created before disabling) then probably it should produce some warning/error in the output informing the user about it being disabled. Just seeing an empty subinspector:
![lHfJEI8THK](https://user-images.githubusercontent.com/9283098/196184288-23b718e7-6340-45b4-8179-aafcedaf71e2.png)
isn't too informative, as this issue shows.
